docs/content/concepts/timelines.md

```diff
@@ -4,24 +4,22 @@ order: 3
 ---
 
 ## Timelines
+
 Each piece of logged data is associated with one or more timelines.
 By default, each log is added to the `log_time` timeline, with a timestamp assigned by the SDK.
 
-In Python, use the _set time_ functions ([set_time_sequence](https://ref.rerun.io/docs/python/stable/common/logging_functions/#rerun.set_time_sequence), [set_time_seconds](https://ref.rerun.io/docs/python/stable/common/logging_functions/#rerun.set_time_seconds), [set_time_nanos](https://ref.rerun.io/docs/python/stable/common/logging_functions/#rerun.set_time_nanos)) to associate logs with other timestamps on other timelines. For example:
-
-```python
-for frame in read_sensor_frames():
-    rr.set_time_sequence("frame_idx", frame.idx)
-    rr.set_time_seconds("sensor_time", frame.timestamp)
-
-    rr.log("sensor/points", rr.Points3D(frame.points))
-```
+You can use the _set time_ functions (Python reference: [set_time_sequence](https://ref.rerun.io/docs/python/stable/common/logging_functions/#rerun.set_time_sequence), [set_time_seconds](https://ref.rerun.io/docs/python/stable/common/logging_functions/#rerun.set_time_seconds), [set_time_nanos](https://ref.rerun.io/docs/python/stable/common/logging_functions/#rerun.set_time_nanos)) to associate logs with other timestamps on other timelines. For example:
 
-<!-- TODO(emilk): add Rust version -->
+code-example: timelines_example
 
 This will add the logged points to the timelines `log_time`, `frame_idx`, and `sensor_time`.
 You can then choose which timeline you want to organize your data along in the expanded timeline view in the bottom of the Rerun Viewer.
 
+## Events
+
+An _event_ refer to an instance of logging one or more component batches to one or more timelines. In the viewer, the Time panel provide a graphical representation of these events across time and entities. 
```

Comment:
A screenshot here showing the timeline, maybe with the event hovered would be nice.